server/applicationPattern/applicationPattern/onfModel/services/LogicalTerminationPointWithMappingServices.js

```diff
@@ -333,7 +333,7 @@ function updateLogicalTerminationPointInstanceGroupAsync(logicalTerminationPoint
             let httpClientUuid = await httpClientInterface.getHttpClientUuidAsync(
                 applicationName
             );
-            tcpClientConfigurationStatusList = await createOrUpdateTcpClientInterface(
+            tcpClientConfigurationStatusList = await createOrUpdateTcpClientInterfaceForRegardApplication(
```

Comment:
This is a common method used by every other functionality . Can we create a method specific to this functionality

Reply:
fixed